app-modules/notifications/src/Models/Contracts/Subscribable.php

```diff
@@ -11,6 +11,4 @@
  *
  * @mixin Model
  */
-interface Subscribable
-{
-}
+interface Subscribable {}
```

Comment:
Do we still need this with the introduction of `CanTriggerAutoSubscription`?

Reply:
We do; this is the interface for the Subscribable Model (i.e. `Student` and `Prospect`) whereas `CanTriggerAutoSubscription` is for the related Models that can trigger an Auto Subscription like `CaseItem`